wp-content/themes/cjet/guides.php

```diff
@@ -137,14 +149,14 @@
 							);
 						} else {
 							if (!empty($prev_id)) {
-								printf( '<div class="previous"><a href="%1$s"><h5>Previous Section</h5><span class="meta-nav">%2$s</span></a></div>',
+								printf( '<div class="previous"><a href="%1$s"><i class="dashicons dashicons-arrow-left-alt"></i><span class="meta-nav">%2$s</span></a></div>',
```

Comment:
should this `<a` be `rel="prev"`?

https://www.w3.org/TR/WCAG20-TECHS/H59.html describes why, https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types says that the `rel=""` attribute can be used on `<a>`.

Reply:
Good call.